python/nano/notebooks/nano-inference-example.ipynb

```diff
@@ -0,0 +1,400 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "10a585bf",
+   "metadata": {},
+   "source": [
+    "## Bigdl-nao onnxruntime example\n",
+    "--- \n",
+    "This example shows the usage of bigdl-nano pytorch onnxtuntime. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "14f6dfab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
```

Comment:
How about just loading the checkpoint generated during the training notebook.

Reply:
OK, I will modify the notebook. Should I put a .ckpt file in the directory？